src/lib.rs

```diff
@@ -1139,6 +1139,21 @@ impl Build {
         for obj in objs {
             self.compile_object(obj)?;
         }
+
+        if self.cuda {
+            let compiler = self.try_get_compiler()?;
+            let mut cmd = compiler.to_command();
+            cmd.arg("-dlink");
+            for obj in objs {
+                cmd.arg(obj.dst.clone());
+            }
+            cmd.arg("-o");
+            let out_dir = self.get_out_dir()?;
+            let out_dir = out_dir.join("link.o");
```

Comment:
Could `"link.o"` shared between here and below be refactored to a common definition? Additionally could it be named something like `__cc_internal_link.o` to try to avoid name clashes with normal objects?

Reply:
Thanks, I renamed to `__cc_internal_link.o`.